CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.0.1]
+### Performance
+* Reduce thread_yield call
```

Comment:
Reduce number of calls, or the cost of the call?

Reply:
Reduce number of calls.